test/e2e/machine_pool_test.go

```diff
@@ -24,7 +24,7 @@ import (
 	. "github.com/onsi/ginkgo"
 )
 
-var _ = Describe("When testing MachinePools [Periodic-K8SVersion]", func() {
+var _ = Describe("When testing MachinePools [Feature]", func() {
```

Comment:
Does this mean that when new tests are added, developers need to add `[Feature]` to it? I noticed that some tests like the node drain timeout or mhc remediations test without any focus flags.
I think this may be a little confusing in the long term. 🤔

Reply:
The long term idea behind this was to give more meaningful names for batching a set of tests. That being said, I did not want to alter the behavior of the current e2e test suite and hence just replaced the label names. Did not add in any new ones.

Reply:
Pr for test infra: https://github.com/kubernetes/test-infra/pull/20720